### ISSUE TITLE
nabc support broken by the renaming

### ISSUE THREAD

**Issue by jakubjelinek** [gregorio-project/gregorio#585]
Starting with 18958c1d89ba4fedfd7b889b8e369c1cc6d949c7 it seems at least with TexLive 2014 the nabc support is completely broken.  Even with the GreScor -> GreScore fix and changes to the GregorioNabcRef.tex file, the documentation only builds if I comment out the inclusion of veni.gtex there.
28e725f2cb21ae46e9d7bcfb9addc90268e2016c builds just fine.
@rpspringuel , the error I'm getting is:
(./veni-4_0_0-beta2.gtex
! Undefined control sequence.
\GreNABCChar ...}\catcode `\@ =11{}\gre@font@nabc 
                                                  \gre@style@nabc \grenabcch...
l.16 }
    %
I fear this is related to @ being different category when \GreNABCNeumes is used, but even including just \GreNABCChar{ta>} instead of \GreNABCNeumes{1}{ta>} doesn't really help.

**Comment by rpspringuel** [comment]
I remember dealing with something like this on Thursday and thought I had fixed it.  I'll check when I get home this afternoon. Perhaps I forgot to upload something.

**Comment by henryso** [comment]
I don't want to step on any toes in case this is in flight; but, for me, the fix is to change ...
```
  \ifgre@nabcfontloaded%
    \gresetnabcfont{\gre@nabcfontname}{\gre@nabcfontsize}%
  \fi %
```
... to ...
```
  \ifgre@nabcfontloaded%
  \else %
    \gresetnabcfont{\gre@nabcfontname}{\gre@nabcfontsize}%
  \fi %
```
... at the end of gregoriotex-nabc.tex

**Comment by rpspringuel** [comment]
Looking at my local repository, I notice that that is exactly the fix I have, but that it's not yet been committed.  Guess I need to fix that.

**Comment by rpspringuel** [comment]
Fixed: e848e6f
